cogstat/cogstat_stat.py

```diff
@@ -641,9 +662,11 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
             r_ci_low, r_ci_high = cs_stat_num.corr_ci(r, df + 2)
             pdf_result.loc[_("Pearson's correlation") + ', <i>r</i>'] = \
                 ['%0.3f' % r, '[%0.3f, %0.3f]' % (r_ci_low, r_ci_high)]
+
+            # Warnings based on the results of the assumption tests
             if normality is None:
-                standardized_effect_size_result += '\n' + '<decision>' + _('Normality could not be calculated.') + \
-                                                   ' ' + _('CIs may be biased.') + '</decision>'
+                standardized_effect_size_result += '\n' + '<decision>' + _('Normality could not be calculated!') \
```

Comment:
We can get back to the original message too.

Reply:
Reverted.